Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -162,6 +162,9 @@ extension DiagnosticMessage where Self == StaticParserError {
   public static var forbiddenInterpolatedString: Self {
     return .init("argument cannot be an interpolated string literal")
   }
+  public static var genericParamCantBeUsedInEnumCaseDecl: Self {
+    return .init("generic signature cannot be declared in enum 'case'. did you mean to attach it to enum declaration?")
```

Comment:
Since we don’t offer a Fix-It

```suggestion
    return .init("generic signature cannot be declared in enum 'case'")
```

Reply:
You still need to update the tests